baseplate/lib/metrics.py

```diff
@@ -242,7 +242,7 @@ def flush(self) -> None:
             )
             logger.warning(
                 "Metrics batch of %d bytes is too large to send, flush more often or reduce "
-                "amount done in this request. Top counters: %s",
+                "amount done in this request. See https://git.io/JJscR. Top counters: %s",
```

Comment:
note: this link won't work until this is merged to master. it's a fun little hack to get a nice git.io shortlink for something not hosted on github.com (eek?): it points here: https://github.com/reddit/baseplate.py/blob/gh-pages/shortlinks/metrics-batch-too-large.html